src/Chapter2.hs

```diff
@@ -336,7 +336,14 @@ from it!
 ghci> :l src/Chapter2.hs
 -}
 subList :: Int -> Int -> [a] -> [a]
-subList = error "subList: Not implemented!"
+subList start end l
+    | start < 0 || end < 0 = []
+    | otherwise =
+        let 
+            takeNumber = end - start + 1;
+        in
+        take takeNumber (drop start l)
```

Comment:
Your implementation is correct when the right bound is lower than the left. But I think you can implement it more efficiently by adding a separate case to your check :slightly_smiling_face: